examples/drones/doc_gen.py

```diff
@@ -3,8 +3,8 @@
 from hydrus.hydraspec.doc_writer import HydraDoc, HydraClass, HydraClassProp, HydraClassOp
 import json
 
-
 def doc_gen(API, BASE_URL):
+    # type: (str, str) -> HydraDoc
```

Comment:
The type notations look fine but currently, these are nothing but comments. You need to implement static type checking and add mypy to the testing environment. 
See https://medium.com/@ageitgey/learn-how-to-use-static-type-checking-in-python-3-6-in-10-minutes-12c86d72677b and http://mypy.readthedocs.io/en/stable/getting_started.html for more info.

Reply:
How should I add mypy to testing environment? As of now, I am running mypy in  hydrus directory with command  ```mypy  .``` to check the type errors. 
Tried mypy-pytest tool but it was too slow.
I think the best way would be to run mypy in the root directory as I am doing before pushing any changes.